src/main/java/io/ebean/config/dbplatform/sqlserver/SqlServerPlatform.java

```diff
@@ -19,23 +24,22 @@
   public SqlServerPlatform() {
     super();
     this.platform = Platform.SQLSERVER;
-    // effectively disable persistBatchOnCascade mode for SQL Server
-    // due to lack of support for getGeneratedKeys in batch mode
-    this.persistBatchOnCascade = PersistBatch.NONE;
     this.idInExpandedForm = true;
     this.selectCountWithAlias = true;
     this.sqlLimiter = new SqlServerSqlLimiter();
     this.basicSqlLimiter = new SqlServerBasicSqlLimiter();
     this.platformDdl = new SqlServerDdl(this);
     this.historySupport = new SqlServerHistorySupport();
-    this.dbIdentity.setIdType(IdType.IDENTITY);
-    this.dbIdentity.setSupportsGetGeneratedKeys(true);
-    this.dbIdentity.setSupportsIdentity(true);
+    dbIdentity.setIdType(IdType.SEQUENCE);
```

Comment:
I Agree, but before introducing a complex migration path, we should ask who already uses sqlserver productive.
As there were so many bugs and also posts on the google groups I think no one uses it really productive, so I would suggest to do a hard switch from IDENTITY to SEQUENCE. For the few pepole this means either to recreate the database or write some custom migration ddls
If most people do not have a problem